config.yaml

```diff
@@ -0,0 +1,420 @@
+---
+imports:
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/iam_service_account_v1/iam_service_account_v1.py
+    name: iam_service_account_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/gce_instancetemplate_v1/gce_instancetemplate_v1.py
+    name: gce_instancetemplate_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/cloudsql_instance_v1/cloudsql_instance_v1.py
+    name: cloudsql_instance_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/address_v1/address_v1.py
+    name: address_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/constants_v1/constants_v1.py
+    name: constants_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/gce_instance_v2/gce_instance_v2.py
+    name: gce_instance_v2.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/ssl_certificate_v1/ssl_certificate_v1.py
+    name: ssl_certificate_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/httphealthcheck_v1/httphealthcheck_v1.py
```

Comment:
Use `compute.v1.healthcheck`, `compute.v1.httphealthcheck`, or `compute.v1.httpshealthcheck` instead of this custom template.

Reply:
Updated